packages/es-client/esAmazonConnection.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const { Connection } = require('@elastic/elasticsearch');
+const aws4 = require('aws4');
```

Comment:
This is probably alright. It's used by 11+ million people. There hasn't been a release in 4 years but it also doesn't pull in dependencies we should be concerned with.

Reply:
It seemed like a small enough library to continue using it _for now_.